content/get-started/introduction/_index.md

```diff
@@ -9,13 +9,14 @@ summary: |
   containerization, beginning with foundational concepts and installation
   procedures. Progress through hands-on exercises that cover essential Docker
   commands, image creation, and container orchestration.
-aliases:
-  - /get-started/
 layout: series
 params:
   skill: Beginner
   time: 15 minutes
   prereq: None
+aliases:
```

Comment:
I can't add a comment on this topic's title frontmatter because it didn't change in this PR, but should it change to match the new section name, "Introduction"?